guava/src/com/google/common/primitives/UnsignedBytes.java

```diff
@@ -77,7 +81,7 @@ private UnsignedBytes() {}
    *
    * @since 6.0
    */
-  public static @NonNegative int toInt(byte value) {
+  public static @NonNegative @SignednessGlb int toInt(@UnknownSignedness byte value) {
```

Comment:
Per the [documentation](https://checkerframework.org/manual/#signedness-checker-annotations), please write `@SignedPositive` rather than `@SignednessGlb`.  Please make this change throughout (and in other pull requests as well).